bootstrap/config/config.go

```diff
@@ -283,9 +302,51 @@ func (cp *Processor) Process(
 		}
 	}
 
+	if remoteIPs != nil {
+		err = applyRemoteIPs(remoteIPs, serviceConfig)
+		if err != nil {
+			return err
+		}
+	}
+
 	return err
```

Comment:
this can just be simple as return call:
```suggestion
	return applyRemoteIPs(remoteIPs, serviceConfig)
```

Reply:
Nope, we don't want to call applyRemoteIPs if `remoteIPs ` is nil

Reply:
i got that, i guess you could move that check `remoteIPs` is nil into that func and just return nil as error in that case.